relay/polkadot/src/lib.rs

```diff
@@ -1666,13 +1669,23 @@ construct_runtime! {
 		Mmr: pallet_mmr = 201,
 		BeefyMmrLeaf: pallet_beefy_mmr = 202,
 
+		#[cfg(feature="zombie-bite-sudo")]
```

Comment:
```suggestion
		#[cfg(feature="zombie-bite-sudo")] // FAIL-CI
```
so that we wont forget to remove it.